### ISSUE TITLE
Why do you plot labels y instead of output?

### ISSUE THREAD

**Issue by hzlajojo** [zabir-nabil/text-image-denoiser#2]
Hi, something interesting here:
https://github.com/zabir-nabil/text-image-denoiser/blob/master/infer_demo.py

When we tried to use your code, we noticed that you plot y. I guess it should be output, right?

-------------------------------------------------
cnt = 0
t = 0
for i, data in enumerate(dataloader, 0):
    # get the inputs; data is a list of [inputs, labels]
    X, y = data
    t1 = time.time()
    X = X.cuda()
    y = y.cuda()
    outputs = model(X)
    t2 = time.time()
    
    t += (t2-t1)

    plot_save(X[0,:,:,:].cpu(), **y[0,:,:,:].cpu()**, 'unet_demo/', cnt)
    
    cnt += 1
    
    if cnt == 100:
        break

print(f'Time average: {t/100.}')

**Comment by zabir-nabil** [comment]
yes, it should be `outputs`, the script is only to measure the approximate inference time.
